app/src/components/Graph.tsx

```diff
@@ -112,6 +112,32 @@ function Graph({
     }
   }, [textToParse]);
 
+  const downloadImageAsPNG = useCallback(() => {
+    if (cy.current) {
+      // @ts-ignore
+      const pngStr = cy.current.png({ full: true, scale: 1.5, output: 'blob' });     
+      saveAs(
+        new Blob([pngStr], {
+          type: "image/png",
+        }),
+        "flowchart.png"
+      );
+    }
+  }, [textToParse]);
```

Comment:
Because `jpg` and `png` are core functions to cytoscape, the `cy.current` ref is enough. We don't need to regenerate these functions when the text changes, so you can remove `textToParse` from the dependency array. That should fix the build as well 👍

Reply:
Didn't know that :)